### PR TITLE
E2167 - Testing Team Related Files

### PR DESCRIPTION
**Pull request by MathewAaron**
Please set the title of your pull request in the following format: `[Project id/Independent study/Issue fix/...]. project name`, eg., `E904. Integrating courses and assignments on Expertiza with Moodle`

We suggest you including the below information in your pull request:
- A description of the changes proposed in the pull request.
- @mentions of the person or team responsible for reviewing proposed changes.
---
About Expertiza Bot
- If you have any questions about comments given by the expertiza-bot [(example)](https://github.com/expertiza/expertiza/pull/1877#issuecomment-762412918), you could create a comment in your pull request with the format `/dispute [UUID1] [UUID2]` [(example)](https://github.com/expertiza/expertiza/pull/1877#issuecomment-762430057) then the professor and TAs will be notified.
- [Here is a video](https://tinyurl.com/internet-bots) about how to communicate to expertiza-bot.

### CODE REVIEW

**Comment by nnhimes** [comment]
Issue comment pasted:

The' update' method fails because the methods called inside 'begin' statements cant be stubbed. The '.save' call on stubbed objects are not successful because stubbed objects can't be saved in the database.

cc: @priyajakhar

**Comment by nnhimes** [comment]
Issue comment pasted:

Among 15 tests the def index method fails due to missing template. I tried to look around for the template so that I could redirect to it but was unnsuccessful.

Rest all the test for this controller passes, with coverage 85.71%

**Comment by johnbumgardner** [comment]
@nnhimes i added an index file. in the controller i added a check to make sure the user had admin privileges to view it. wrote up some corresponding spec tests

**Review by johnbumgardner — APPROVED** [review]

File: db/schema.rb [hunk 1]
```diff
@@ -30,8 +30,9 @@
     t.integer  "tag_prompt_deployment_id", limit: 4
     t.integer  "user_id",                  limit: 4
     t.string   "value",                    limit: 255
-    t.datetime "created_at",                           null: false
-    t.datetime "updated_at",                           null: false
+    t.datetime "created_at",                                                    null: false
+    t.datetime "updated_at",                                                    null: false
+    t.decimal  "confidence_level",                     precision: 10, scale: 5
```
Comment (nnhimes) [inline db/schema.rb:35]: Why is line 35 being added? I can't find another place confidence_level is used in this PR

**Pull request merged**